src/main/java/com/spotinst/sdkjava/example/admin/organization/UserManagementUsageExample.java

```diff
@@ -280,4 +288,68 @@ private static Boolean deleteOrganization(SpotinstAdminOrganizationClient adminC
 
         return status;
     }
+
+    private static String createAccessPolicy(SpotinstAdminOrganizationClient adminClient) {
+        List<String> actions = new ArrayList<>();
+        actions.add("*");
```

Comment:
Can we take some more specific example in actions and policy? Just * will not be helpful to the customers referring these examples.